tests/tests.py

```diff
@@ -235,7 +235,7 @@ def test_this(self):
     def test_dow(self):
         now = datetime.now()
         hour_in_seconds = 24 * 60 * 60
-        for x, day in enumerate(('monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday')):
+        for x, day in enumerate(('monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'Saturday', 'sunday')):
```

Comment:
any reason the we are capitalizing just Saturday?

Reply:
Iirc,  a leftover typo from Steve.